admin/app/controllers/spree/admin/digitals_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Spree
+  module Admin
+    class DigitalsController < ResourceController
```

Comment:
@Vegann let's rename this to `DigitalAssets` please, the short-hand version is kinda confusing